frontend/www/js/omegaup/group/edit.ts

```diff
@@ -296,29 +298,92 @@ OmegaUp.on('ready', () => {
       }),
     );
   }
+});
 
-  function generatePassword(): string {
-    const validChars = 'acdefhjkmnpqruvwxyACDEFHJKLMNPQRUVWXY346';
-    const len = 8;
-    // Browser supports window.crypto
-    if (typeof window.crypto == 'object') {
-      const arr = new Uint8Array(2 * len);
-      window.crypto.getRandomValues(arr);
-      return Array.from(
-        arr.filter((value) => value <= 255 - (255 % validChars.length)),
-        (value) => validChars[value % validChars.length],
-      )
-        .join('')
-        .substr(0, len);
-    }
+export function generatePassword(humanReadable: boolean): string {
+  if (humanReadable) {
+    const words = {
+      es: [
+        'Loro',
+        'Perro',
+        'Pollo',
+        'Lagarto',
+        'Gato',
+        'Toro',
+        'Vaca',
+        'Sapo',
+        'Oso',
+        'Zorro',
+      ],
+      en: [
+        'Parrot',
+        'Dog',
+        'Chicken',
+        'Lizard',
+        'Cat',
+        'Bull',
+        'Cow',
+        'Frog',
+        'Bear',
+        'Fox',
+      ],
+      pt: [
+        'Papagaio',
+        'Cão',
```

Comment:
```suggestion
        'Cachorro',
```

así no hay diacríticos